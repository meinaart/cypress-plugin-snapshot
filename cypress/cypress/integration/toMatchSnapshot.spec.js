describe('toMatchSnapshot', () => {
  it('toMatchSnapshot - json', () => {
    cy.request('/static/stub.json')
      .its('body')
      .toMatchSnapshot();
  });

  it('toMatchSnapshot - html', () => {
    cy.visit('/static/stub.html')
      .then(() => {
        cy.get('[data-test=test]').toMatchSnapshot();
      });
  });
});
