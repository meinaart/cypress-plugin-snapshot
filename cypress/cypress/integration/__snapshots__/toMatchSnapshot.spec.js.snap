exports[`toMatchSnapshot > toMatchSnapshot - json #0`] = 
{
  "number": 1234,
  "object": {
    "foo": "bar",
    "nestedObject": {
      "bar": "foo"
    }
  },
  "string": "string"
};

exports[`toMatchSnapshot > toMatchSnapshot - html #0`] = `
<div class="wrapper" data-test="test">
    <h1>Lorem Ipsum</h1>
    <p>Lorem ipsum dolor sit amet, <a href="#">consectetur</a> adipiscing elit. Integer dignissim, odio sed interdum porta, urna ex ultricies tellus, non fringilla purus nulla nec augue. Vivamus eget iaculis lorem. Proin blandit dictum enim, vitae efficitur arcu euismod a. Sed sodales nisi volutpat dui convallis ullamcorper. Aenean eget semper metus. Curabitur quis imperdiet lorem. Ut vitae mauris augue. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut rhoncus augue et scelerisque malesuada. Pellentesque bibendum sit amet tellus sit amet pellentesque. Maecenas eu magna dapibus, tristique est nec, ornare libero. Sed consectetur consequat nisi.</p>
    <h2>Curabitur finibus quam</h2>
    <p>Curabitur finibus quam quam, in viverra nulla dapibus sed. Praesent cursus nec erat ut tempor. Donec pharetra rutrum aliquam. Mauris eget gravida quam. Aenean sodales eros a <a href="#">felis lobortis</a> faucibus. Donec sed luctus purus. Nullam nisi ipsum, scelerisque at felis eu, aliquam condimentum ex. Quisque nec orci at justo faucibus maximus. Suspendisse lobortis id mauris in pellentesque. Praesent et dolor ultricies, vestibulum ipsum vel, auctor lacus. Nullam nec dui id arcu consectetur aliquam.</p>
  </div>
`;

exports[`Test > toMatchSnapshot - json #0`] = 
{
  "number": 1234,
  "object": {
    "foo": "bar",
    "nestedObject": {
      "bar": "foo"
    }
  },
  "string": "string"
};

exports[`Test > toMatchSnapshot - html #0`] = `

<div data-test="test">
  <h1>Title</h1>
  <p>Paragraph 1</p>
  <p>Paragraph 2</p>
</div>

`;
